packages/webview_flutter/example/lib/main.dart

```diff
@@ -47,3 +62,40 @@ class SampleMenu extends StatelessWidget {
     );
   }
 }
+
+class MoveControl extends StatelessWidget {
```

Comment:
nit: "Move" doesn't sound about right here, maybe call this widget: `NavigationControls` ?

Reply:
Fixed on 88f299b